apps/nextjs-website/src/lib/api.ts

```diff
@@ -8,14 +8,22 @@ export function getOverviewPaths(): readonly string[] {
   return overviews.map((overview) => overview.path);
 }
 
+export function getApisPaths(): readonly string[] {
+  return apis.map((api) => api.path);
+}
+
 export function getGuides(productSlug?: string) {
   return productsGuides.find(
-    (productGuides) => productGuides.product.path === `/${productSlug}`
+    (productGuides) => productGuides.product.path === productSlug
   );
 }
 
 export function getOverview(productSlug?: string) {
   return overviews.find(
-    (overviewData) => overviewData.product.path === `/${productSlug}`
+    (overviewData) => overviewData.product.path === productSlug
```

Comment:
I think this change is wrong. Probably due to a wrong merge with the branch main